config/groovy/common.groovy

```diff
@@ -182,12 +182,47 @@ class common {
         addRemote(itemName, defaultRemote, "https://github.com/${githubDefaultHome}/${itemName}.git")
     }
 
+    /**
+     * Check if an item was updated within the provided time limit
+     * @param fetchHead the item's FETCH_HEAD file in the .git directory
+     * @param timeLimit the time limit for considering something recently updated, for example: use(groovy.time.TimeCategory){ 10.minute }
+     */
+    def isRecentlyUpdated(File fetchHead, def timeLimit){
+        Date lastUpdate = new Date(fetchHead.lastModified())
+        def recentlyUpdated = use(groovy.time.TimeCategory){
+            def timeElapsedSinceUpdate = new Date() - lastUpdate
+            if (timeElapsedSinceUpdate < timeLimit){
+                return true
+            } else {
+                return false
+            }
+        }
+    }
+
     /**
      * Update a given item.
      * @param itemName the name of the item to update
      */
-    def updateItem(String itemName) {
+    def updateItem(String itemName, boolean skipRecentUpdates = false) {
         File targetDir = new File(targetDirectory, itemName)
+        def searchString = itemName
```

Comment:
What is this declaration used for? Are you modifying the `searchString` somewhere?

Reply:
This accidentally returned when I reverted a commit. At one point I was modifying the search string to find relative paths... I cut this out